.github/workflows/publish-images.yaml

```diff
@@ -0,0 +1,118 @@
+  push:
+    name: Push images
+    environment: Release
+    needs: [prepare,build]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: [amd64, arm64]
+        registry: [rhcc, gcr, dockerhub]
+        include:
+          - registry: rhcc
+            url: registry.connect.redhat.com
+            repository: dynatrace/dynatrace-operator
+            username: ${{ secrets.RHCC_USERNAME }}
+            password: ${{ secrets.RHCC_PASSWORD }}
+          - registry: gcr
+            url: gcr.io
+            repository: dynatrace-marketplace-prod/dynatrace-operator
+            username: ${{ secrets.GCR_USERNAME }}
+            password: ${{ secrets.GCR_PASSWORD }}
+          - registry: dockerhub
+            url: docker.io
+            repository: dynatrace/dynatrace-operator
+            username: ${{ secrets.DOCKERHUB_USERNAME }}
+            password: ${{ secrets.DOCKERHUB_PASSWORD }}
+    steps:
+      - name: Push ${{matrix.platform}} to ${{matrix.registry}}
+        uses: ./.github/workflows/upload-dockerimage.yaml
+        with:
+          platform: ${{matrix.platform}}
+          labels: ${{ needs.prepare.outputs.labels }}
+          version: ${{ needs.prepare.outputs.version }}
+          registry: ${{matrix.url}}
+          repository: ${{matrix.repository}}
+          secrets.docker_repo_username: ${{matrix.username}}
+          secrets.docker_repo_password: ${{matrix.password}}
+
+  manifest:
+    name: Create manifests
+    needs: [prepare, push]
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        registry: [ rhcc, gcr, dockerhub ]
+        include:
+          - registry: rhcc
+            url: registry.connect.redhat.com
+            repository: dynatrace/dynatrace-operator
+            username: ${{ secrets.RHCC_USERNAME }}
+            password: ${{ secrets.RHCC_PASSWORD }}
+          - registry: gcr
+            url: gcr.io
+            repository: dynatrace-marketplace-prod/dynatrace-operator
+            username: ${{ secrets.GCR_USERNAME }}
+            password: ${{ secrets.GCR_PASSWORD }}
+          - registry: dockerhub
+            url: docker.io
+            repository: dynatrace/dynatrace-operator
+            username: ${{ secrets.DOCKERHUB_USERNAME }}
+            password: ${{ secrets.DOCKERHUB_PASSWORD }}
+    steps:
+      - name: Login to ${{matrix.repository}}
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ matrix.url }}
+          username: ${{ matrix.username }}
+          password: ${{ matrix.password }}
+      - name: Create manifest for ${{matrix.repository}}
+        env:
+          IMAGE: ${{matrix.url}}/${{matrix.repository}}
+        # Simply pulls, otherwise the artifact would need to be re-downloaded and re-tagged correctly which would mean
+        # two more steps instead of two more lines
+        run: |          
+          docker pull ${{matrix.url}}/${{matrix.repository}}:${{ needs.prepare.outputs.version }}-amd64
```

Comment:
```suggestion
          docker pull ${{IMAGE}}:${{ needs.prepare.outputs.version }}-amd64
```

Reply:
why the pull? does the image need to be in the local registry?

Reply:
can manifests be created without having the images locally?